src/game/AI/EventAI/CreatureEventAI.cpp

```diff
@@ -1658,6 +1657,8 @@ void CreatureEventAI::EnterCombat(Unit* enemy)
     }
     ProcessEvents(enemy);
 
+    CreatureAI::EnterCombat(enemy);
```

Comment:
No. A lot implicit behaviour in old scripts relies on this never changing. What had to move was the flee. Alrdy done that separately.